ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py

```diff
@@ -416,3 +416,33 @@ def test_linear(
         weight=weight,
         bias=bias,
     )
+
+#unsample
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.common.unsample",
+    dtype_x_weight_bias=x_and_linear(
+        dtypes=helpers.get_dtypes("float", full=False),
+    ),
+)
+def test_unsample(
+    *,
+    dtype_x_weight_bias,
+    on_device,
+    fn_tree,
+    backend_fw,
+    frontend,
+    test_flags,
+):
+    dtype, x, weight, bias = dtype_x_weight_bias
+    weight = ivy.swapaxes(weight, -1, -2)
```

Comment:
Also we don't want `Ivy` dependency in unit tests we aim to be using numpy here only.